contracts/modules/capital/Ramm.sol

```diff
@@ -329,6 +371,10 @@ contract Ramm is IRamm, MasterAwareV2 {
     return 1 ether * capital / supply;
   }
 
+  function isSwapPause() public view returns (bool) {
+    return swapPaused;
+  }
```

Comment:
To be inline with NXMMaster which has a `isPause` view function, I wrote `isSwapPause` here.

But not sure if this function is necessary as we can just read the public `swapPaused` variable.

Also I'd prefer it with a `d` - isSwapPause**d** but I'm choosing to prioritise consistency with existing code